src/app/content/components/BookBanner.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import withServices from '../../context/Services';
+import styled from 'styled-components';
+import theme from '../../theme';
+import { AppServices, AppState } from '../../types';
+import * as select from '../selectors';
+import { Book, Page } from '../types';
+import {ChevronLeft} from 'styled-icons/boxicons-regular/ChevronLeft'
+import { findArchiveTreeSection } from '../utils'
+import Color from 'color';
+import typography from '../../components/Typography';
+
+const LeftArrow = styled(ChevronLeft)`
+  height: 2rem;
+  width: 2rem;
+  color: #FFFFFF;
+  margin-right: 7px;
+`;
+
+interface PropTypes {
+  page?: Page;
+  book?: Book;
+  services: AppServices;
+}
+
+const TopBar = styled.div`
+  height: 13rem;
+  width: 100%;
+  max-width: 1170px;
+  font-size: 1.25rem;
+  font-weight: bold;
```

Comment:
do you need these font styles on the whole bar? don't you just override them in the text elements?